### ISSUE TITLE
Customize Task UUID Symbol

### ISSUE THREAD

**Issue by shfattig** [tools-life/taskwiki#436]
Currently, TaskWiki shows the UUID of tracked tasks by appending `  #{uuid}` to the end of the todo item. Unfortunately this conflicts with tools which use `#` to denote tags, with a massive list of uuid junk presented on tag searches.

It would be very helpful if the user could deconflict this syntax by customizing the symbol used to something other than `#` -- perhaps `${uuid}` or `%{uuid}` or `|{uuid}`

**Comment by shfattig** [comment]
I've implemented a working solution that handles (almost) every symbol and created a pull request #437. There are a few things that I'd love @maintainers help and insight on, so please review and comment!

**Comment by Archie-2021** [comment]
Any update on this, I just finish set it up to work with obsidian, and Obsidian recognized in test `#` as hashtag, I think using something like `^` for the marker is the more sensible option to use for default

**Comment by shfattig** [comment]
Hi @Archie-2021 ! Good question - no progress. I never heard anything from maintainers, but maybe if the issue or MR gets more attention they may offer input? I've just been using my own tweaks on my own computer but I'd love to ship it as an official feature of the project!

**Comment by Archie-2021** [comment]
> Hi @Archie-2021 ! Good question - no progress. I never heard anything from maintainers, but maybe if the issue or MR gets more attention they may offer input? I've just been using my own tweaks on my own computer but I'd love to ship it as an official feature of the project!

Hi

Interesting , i see a repo in you github related to taskwiki, but could not find anything different, is that the repo you are talking about?
I like to take a look, i am new to this workflow so I am open to new ideaa

**Comment by shfattig** [comment]
Yeah! The changes are shown in merge request #437, mentioned in a message above

**Comment by FelipeMarcelino** [comment]
Would be good if the PR got merged, because most of the tools recognize # as hashtag.
